exporter/otlpexporter/otlp.go

```diff
@@ -103,18 +105,21 @@ func (e *exporter) shutdown(context.Context) error {
 func (e *exporter) pushTraces(ctx context.Context, td ptrace.Traces) error {
 	req := ptraceotlp.NewRequestFromTraces(td)
 	_, err := e.traceExporter.Export(e.enhanceContext(ctx), req, e.callOptions...)
+	e.settings.Logger.Debug("Exporting traces for ", zap.Stringer("traceId", trace.SpanContextFromContext(ctx).TraceID()))
```

Comment:
```suggestion
	e.settings.Logger.Debug("Exporting traces", zap.Stringer("traceId", trace.SpanContextFromContext(ctx).TraceID()))
	req := ptraceotlp.NewRequestFromTraces(td)
	_, err := e.traceExporter.Export(e.enhanceContext(ctx), req, e.callOptions...)
```

I would emit these logs at the beginning of these methods so that it is obvious that exporting has started even if it takes some time.  Also, the message should be able to stand alone; the additional attributes are to add context and are not part of the log message.

Reply:
Okay, will do.  thanks @Aneurysm9